antarest/study/business/model/link_model.py

```diff
@@ -78,6 +86,11 @@ def to_internal(self, version: StudyVersion) -> "LinkInternal":
         return LinkInternal(**data)
 
 
+class LinkDtoForUpdate(LinkDTO):
+    area1: SkipJsonSchema[str] = Field("a", exclude=True)
```

Comment:
Weird way to do it. Why did you introduce fields `a` and `b` ?

Reply:
After testing this allows user to specfiy area1 and area2 inside the body and they we'll be ignored. I don't think we want that. I'd prefer you introduce a class that just has the base attributes and make LinkDTO inherit from this :/

Reply:
I changed it